docs/po/pt.po

```diff
@@ -1590,6 +1590,1002 @@ msgstr "`/blockhash/<ALTURA>`: hash do bloco em uma determinada altura de bloco.
 msgid "`/blocktime`: UNIX time stamp of latest block."
 msgstr "`/blocktime`: marca de tempo UNIX do bloco mais recente."
 
+#: src/runes.md:4
+msgid "Runes allow Bitcoin transactions to etch, mint, and transfer Bitcoin-native digital commodities."
+msgstr "As runes permitem que as transações de Bitcoin gravem, mintem e transfiram artefatos digitais nativas do Bitcoin."
+
+#: src/runes.md:7
+msgid "Whereas every inscription is unique, every unit of a rune is the same. They are interchangeable tokens, fit for a variety of purposes."
+msgstr "Enquanto cada inscrição é única, cada unidade de uma rune é a mesma. São tokens intercambiáveis, adequados para uma variedade de propósitos."
+
+#: src/runes.md:10 src/runes/specification.md:20
+msgid "Runestones"
+msgstr "Pedras Rúnicas (Runestones)"
+
+#: src/runes.md:13
+msgid "Rune protocol messages, called runestones, are stored in Bitcoin transaction outputs."
+msgstr "As mensagens do protocolo rune, chamadas runestones, são armazenadas em resultados de transação no Bitcoin."
+
+#: src/runes.md:16
+msgid ""
+"A runestone output's script pubkey begins with an `OP_RETURN`, followed by `OP_13`, followed by zero or more data pushes. These data pushes are concatenated and decoded into a "
+"sequence of 128-bit integers, and finally parsed into a runestone."
+msgstr ""
+"O output da pubkey de uma runestone começa com `OP_RETURN`, seguido por `OP_13`, seguido por zero ou mais dados inseridos. Estes dados são concatenados e "
+"descodificados numa sequência de números inteiros de 128 bits e finalmente transformados numa runestone."
+
+#: src/runes.md:21
+msgid "A transaction may have at most one runestone."
+msgstr "Uma transação pode ter, no máximo, uma runestone."
+
+#: src/runes.md:23
+msgid "Uma runestone pode gravar uma nova rune, mintar uma rune existente, e transferir runes das entradas (input) de uma transação para as suas saídas (outputs)."
```

Comment:
Hey friend. I think the msgid is in english, right? You replaced in the wrong place :)

Reply:
Cheers Danilo. On my github it was in english but it was missing a space. should be good now 👍